app/controllers/account/activities_controller.rb

```diff
@@ -4,6 +4,7 @@ class Account::ActivitiesController < ApplicationController
 
   def show
     @logs = current_user.event_logs.page(params[:page]).per(100)
+    @user = current_user
```

Comment:
Now we're calling `current_user` directly in the view, we're not using this instance variable anymore, so it can be removed.